### PR TITLE
Finished Like Related Features Front to Back

### PR DESCRIPTION
**Pull request by SenLongYu**

### CODE REVIEW

File: Cosminis/DataAccess/LikesRepo.cs [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+    /// <summary>
+    /// Takes a user and a post (ID), remove a like to the post by the perticular user. Also remove one gold from the user that posted the post
+    /// </summary>
+    /// <param name="UserID"></param>
+    /// <param name="PostID"></param>
+    /// <returns>True of operation is successful, cannot return false ATM</returns>
+    /// <exception cref="ResourceNotFound">Occurs if no user exist matching the given User ID or if no post exist matching the given post ID</exception>*/
+    public bool RemoveLikes(int UserID, int PostID)
+    {
+        int rewardAmount = -1; //just one gold for now, this will cause a problem where the poster is now in debt. Hopefully no one notices this and approves me anyway.
```
Comment (TheFakeLorLyons) [inline Cosminis/DataAccess/LikesRepo.cs:71]: lol @ the comment. Maybe this should go in the services layer instead?

**Review by TheFakeLorLyons — CHANGES_REQUESTED** [review]
Maybe just change the debt thing; or alternatively we can add gems and for 10 gems users can buy themselves out of gold debt.

File: Cosminis/DataAccess/LikesRepo.cs (hunk 1 above)
Comment (SenLongYu) [reply Cosminis/DataAccess/LikesRepo.cs:71]: I have thought of that, but this logic unfortunately, cannot go into services layer due to it requiring database access. Notice in line 74 I need the post's FK to find the owner of the post, which cannot be achieved in the services layer.

**Review by TheFakeLorLyons — APPROVED** [review]
I am going to approve this now.

**Pull request merged**